scripts/sct_label_utils.py

```diff
@@ -95,10 +95,13 @@ def get_parser():
         '-create-seg',
         metavar=Metavar.list,
         type=list_type(':', list_type(',', int)),
-        help="R|Create labels along cord segmentation (or centerline) defined by '-i'. First value is 'z', second is "
-             "the value of the label. Separate labels with ':'. Example: 5,1:14,2:23,3. \n"
-             "To select the mid-point in the superior-inferior direction, set z to '-1'. For example if you know that "
-             "C2-C3 disc is centered in the S-I direction, then enter: -1,3"
+        help="R|Create labels on a cord segmentation (or centerline) image defined by '-i'. Each label should be "
+             "specified using the form 'v1,v2' where 'v1' is value of the slice index along the superior-inferior "
```

Comment:
there was a mistake before: images are reoriented to RPI, meaning that the 3rd dimension is ordered inferior --> superior (not the other way around). E.g., (x, y, 0) will correspond to the most inferior slice.
```suggestion
             "specified using the form 'v1,v2' where 'v1' is value of the slice index along the inferior-superior "
```

Reply:
Ah, thank you. I'll update all instances.